src/launcher/java/org/truffleruby/launcher/options/CommandLineParser.java

```diff
@@ -84,8 +84,8 @@ public static void processEnvironmentVariable(
             boolean rubyOpts) throws CommandLineException {
 
         String value = System.getenv(name);
-        if (value != null && value.length() != 0) {
-            List<String> args = Arrays.asList(value.split("\\s+"));
+        if (value != null && value.trim().length() != 0) {
```

Comment:
Can we trim above instead of calling `trim()` twice?

Reply:
Yes, fixed.